thorlcr/graph/thgraph.cpp

```diff
@@ -2689,41 +2686,39 @@ class CThorContextLogger : implements IContextLogger, public CSimpleInterface
     }
     virtual void setGlobalId(const char *id, SocketEndpoint &ep, unsigned pid) override
     {
-        globalId.set(id);
-        appendGloballyUniqueId(localId.clear());
+        logTrace.setGlobalId(id);
     }
     virtual void setCallerId(const char *id) override
     {
-        callerId.set(id);
+        logTrace.setCallerId(id);
     }
     virtual const char *queryGlobalId() const
     {
-        return globalId.get();
+        return logTrace.queryGlobalId();
     }
     virtual const char *queryLocalId() const
     {
-        return localId.str();
+        return logTrace.queryLocalId();
     }
     virtual const char *queryCallerId() const override
     {
-        return callerId.str();
+        return logTrace.queryCallerId();
     }
     virtual void setHttpIdHeaders(const char *global, const char *caller)
     {
         if (global && *global)
```

Comment:
minor: logTrace.setHttpIdHeaderNames()...  hopefully to be deleted soon.
Same in a few other places.